packages/config/config/devices/0x0154/flow_stop.json

```diff
@@ -1,25 +1,24 @@
-// Popp & Co 009501
-// Popp 09501 Flow Stop
+// Popp & Co Flow Stop
+// Version 1 & 2
 {
 	"manufacturer": "Popp & Co",
 	"manufacturerId": "0x0154",
-	"label": "009501",
-	"description": "Popp 09501 Flow Stop",
+	"label": "Flow Stop",
+	"description": "Popp Flow Stop Version 1 & 2",
```

Comment:
The description isn't meant to just repeat the manufacturer and label. If there's nothing to add, remove it.